pages/hackrplay/2023/home.js

```diff
@@ -0,0 +1,208 @@
+import { useEffect } from "react";
```

Comment:
Change the directory name and structure based on the suggestion in the description of the issue #109